src/arch/riscv/traps.S

```diff
@@ -115,14 +115,19 @@ handle_syscall:
 
 #ifdef CONFIG_FASTPATH
   li t3, SYSCALL_CALL
-  beq a7, t3, c_handle_fastpath_call
+  bne a7, a3, .Ltest_replyrecv
```

Comment:
Should these changes be `t3` instead of `a3`?

Reply:
Opps, fixed